packages/components/src/custom-select-control/test/index.js

```diff
@@ -87,4 +88,127 @@ describe( 'CustomSelectControl', () => {
 			screen.getByRole( 'button', { name: 'Custom select' } )
 		).toHaveTextContent( 'Hint' );
 	} );
+
+	it( 'Initial value should be replaced when a new item is selected', async () => {
+		const user = userEvent.setup();
+		const mock = jest.fn();
```

Comment:
Providing the mock with a more specific name can be helpful. That way the reader doesn't have to ask "what is this actually mocking". Something like `onChangeMock` works IMHO.

Reply:
Good point!